tck/protocol.py

```
@@ -0,0 +1,119 @@
+    Args:
+        params: Request parameters (dict, list, or None)
+        
+    Returns:
+        Session ID if present, None otherwise
+    """
+    if isinstance(params, dict):
+        return params.get('sessionId')
+    return None
+
+
+def parse_json_rpc_request(request_in: Any) -> Union[Dict[str, Any], JsonRpcError]:
```

Ruff (0.14.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `request_in`

(ANN401)